src/Results.jsx

```diff
@@ -1,8 +1,8 @@
-import Pet from './Pet'
+import Pet from "./Pet";
 
 const Results = ({ pets }) => {
   return (
-    <div className="search">
+    <div className="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-3">
```

Comment:
Here is the magic! We can define how many items will be shown on the each row on xsmall(by declearing `grid-cols-1`), small, and large screens.